modular_skyrat/modules/spacepods/code/spacepods/spacepod_prebuilt.dm

```diff
@@ -0,0 +1,148 @@
+/obj/spacepod/prebuilt
```

Comment:
alright, im gonna say the meme:

why isn't this a vehicle?

Reply:
i am so angry